### PR TITLE
LPS-121553 Tests for Balloon Editor in Balloon Editor Sample

### PR DESCRIPTION
**Pull request by magjed4289**
https://issues.liferay.com/browse/LPS-121553

### CODE REVIEW

**Comment by magjed4289** [comment]
ci:test:frontend

**Review by john-co — APPROVED** [review]
Approved, please see changes. As long as next test run has no unique issues, it's good to forward.

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/ckeditorsample/CKEditorBalloonEditorSample.path [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+<tr>
+	<td>PARAGRAPH_ASSERT_BODY_TEXT_CONTENT_UNDERLINE</td>
+	<td>//div[(@class='portlet-body')]//p//u[contains(text(),'${key_textName}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PARAGRAPH_CONTENT_NUMBERED_LIST</td>
+	<td>//div[(@class='portlet-body')]//ol/li</td>
+	<td></td>
+</tr>
```
[the remark this answers is not in the document]
Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/paths/portlet/ckeditorsample/CKEditorBalloonEditorSample.path]: please see refactor commits.

**Comment by john-co** [comment]
ci:stop

**Comment by john-co** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:frontend:nocompile

**Pull request closed without merging**